store/tikv/lock_resolver.go

```diff
@@ -283,10 +283,12 @@ func (lr *LockResolver) BatchResolveLocks(bo *Backoffer, locks []*Lock, loc Regi
 //    commit status.
 // 3) Send `ResolveLock` cmd to the lock's region to resolve all locks belong to
 //    the same transaction.
-func (lr *LockResolver) ResolveLocks(bo *Backoffer, callerStartTS uint64, locks []*Lock) (int64, error) {
+// The second return value of `pushed` is the lock owner transactions whose
+// minCommitTS have been pushed.
+func (lr *LockResolver) ResolveLocks(bo *Backoffer, callerStartTS uint64, locks []*Lock) (int64, []uint64 /*pushed*/, error) {
```

Comment:
I think it is not necessary to add the comment for the return value in this line.

Reply:
Done... I add the `/*pushed*/` back